### ISSUE TITLE
Integration with Deep Learning libs

### ISSUE THREAD

**Issue by yazun** [kelseyhightower/nocode#718]
Should be added at some point. We are one of TOP500 companies and willing to do this shortly.

**Comment by gjevardat** [comment]
Indeed. It is important to validate that we learn nothing by training nothing with nothing.

**Comment by rdev** [comment]
I was able to integrate TensorNo seamlessly using this:

```










```
It's a little long, but it's worth it

**Comment by yazun** [comment]
Inspirational tensory experience. Will deploy it everywhere here.
